receiver/sshcheckreceiver/metadata.yaml

```diff
@@ -0,0 +1,42 @@
+name: sshcheckreceiver
+
+resource_attributes:
+
+attributes:
+  ssh.endpoint:
+    description: Full SSH endpoint.
+    type: string
+
+metrics:
+  sshcheck.success:
```

Comment:
Based on suggestions and [discussion in httpcheck](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/14191#discussion_r987393250) of @jmacd and @djaglowski I plan to change how we model this to a non-monotonic cumulative sum.